src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+:root {
+  --colour: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  align-self: center;
+}
+
+.logo_link {
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  cursor: pointer;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav_link {
+  display: flex;
+  color: black;
+  text-decoration: none;
+  align-items: center;
+  height: 60px;
+  text-align: center;
+  text-transform: uppercase;
+}
+
+.is-active ,
+.nav_link:hover {
+  color: var(--colour);
+  position: relative;
```

Comment:
Don't alter the position based on the modifier

Better move `position:relative` to `nav_link`

Altering position like this may break layout and is ineffective